prosemirror-jdita/src/tests/document.spec.ts

```diff
@@ -102,7 +102,216 @@ describe('Function unTravel()', () => {
       result = unTravel(doc);
       assert.deepEqual(result, expected);
     });
+
+    it.skip('handles simple jdita document', async () => {
+      const inputXml = `<?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE topic PUBLIC "-//OASIS//DTD LIGHTWEIGHT DITA Topic//EN" "lw-topic.dtd">
+      <topic id="program">
+        <title>Test File 2</title>
+        <body>
+          <section>
+            <p>A test paragraph.</p>
+          </section>
+        </body>
+      </topic>`;
+
+      // original jdita to compare against
+      const originalJdita = await xditaToJson(inputXml);
+
+      // process the jdita document and do the round trip
+      //clean the attributes from the top node to compare
+      originalJdita.attributes = {};
+      // transform the jdita document
+      const transformedJdita = document(originalJdita);
+      // untransform the transformed jdita document
+      const result = unTravel(transformedJdita);
+
+      //compare the original jdita with the result
+      expect(result).to.deep.equal(originalJdita);
+    });
+
+    it.skip('handles complex jdita document', async () => {
+      const inputXml = `<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@marmoure Could you please replace the content of `inputXml` with the constant `XML` in test-utils.ts?  See https://github.com/evolvedbinary/prosemirror-jdita/blob/feature/import-and-save-file/prosemirror-jdita/src/tests/test-utils.ts#L27-L90 
The code will be easier to read.

Reply:
@plutonik-a I used multiple examples, I don't think i can rely on the constant `XML` should we be making more constants?

Reply:
@marmoure Sure, for the reading convenience, you could move all xml example code into test-utils.ts. 
The small examples don't hurt inside the test file from my point of view, but I thought the huge document is just too much.